src/platform/platform_winuser.c

```diff
@@ -67,6 +68,14 @@ QuicProcessorInfoInit(
     uint8_t* Buffer = NULL;
     uint32_t Offset;
 
+    //
+    // dwNumberOfProcessors is specified to be number of processors in current
+    // group. Processor groups are specified to be equal length.
+    //
+    SYSTEM_INFO SysInfo;
+    GetSystemInfo(&SysInfo);
+    QuicProcessorsPerGroup = (uint8_t)SysInfo.dwNumberOfProcessors;
```

Comment:
Can you not get this info from the data below?

Reply:
I'd have to count it manually, to find the count of group 0. I could do that if you want though, but would be a little wonky

Reply:
Ok. I guess I can live with this then.